src/accidental.js

```diff
@@ -422,44 +448,53 @@ export class Accidental extends Modifier {
   setAsCautionary() {
     this.cautionary = true;
     this.render_options.font_scale = 28;
-    this.paren_left = Flow.accidentalCodes("{");
-    this.paren_right = Flow.accidentalCodes("}");
-    var width_adjust = (this.type == "##" || this.type == "bb") ? 6 : 4;
+    this.parenLeft = Flow.accidentalCodes('{');
+    this.parenRight = Flow.accidentalCodes('}');
+    const widthAdjust = (this.type === '##' || this.type === 'bb') ? 6 : 4;
 
     // Make sure `width` accomodates for parentheses.
-    this.setWidth(this.paren_left.width + this.accidental.width + this.paren_right.width - width_adjust);
+    this.setWidth(
+      this.parenLeft.width
+      + this.accidental.width
+      + this.parenRight.width
+      - widthAdjust
+    );
+
     return this;
   }
 
   // Render accidental onto canvas.
   draw() {
-    if (!this.context) throw new Vex.RERR("NoContext",
-      "Can't draw accidental without a context.");
-    if (!(this.note && (this.index != null))) throw new Vex.RERR("NoAttachedNote",
-      "Can't draw accidental without a note and index.");
+    if (!this.context) {
+      throw new Vex.RERR('NoContext', "Can't draw accidental without a context.");
+    }
+
+    if (!(this.note && (this.index != null))) {
+      throw new Vex.RERR('NoAttachedNote', "Can't draw accidental without a note and index.");
+    }
 
     // Figure out the start `x` and `y` coordinates for this note and index.
-    var start = this.note.getModifierStartXY(this.position, this.index);
-    var acc_x = ((start.x + this.x_shift) - this.width);
-    var acc_y = start.y + this.y_shift;
-    L("Rendering: ", this.type, acc_x, acc_y);
+    const start = this.note.getModifierStartXY(this.position, this.index);
+    let accX = ((start.x + this.x_shift) - this.width);
+    const accY = start.y + this.y_shift;
+    L('Rendering: ', this.type, accX, accY);
 
     if (!this.cautionary) {
       // Render the accidental alone.
-      Glyph.renderGlyph(this.context, acc_x, acc_y,
+      Glyph.renderGlyph(this.context, accX, accY,
```

Comment:
I'm okay with argument-per-line, but I don't get the additional level of indentation. It hurts my eyes :-)